From 248b1dbae8b91da5f207f4d6948d6acc90a97e6e Mon Sep 17 00:00:00 2001
From: buffalu <85544055+buffalu@users.noreply.github.com>
Date: Sun, 21 Jul 2024 10:23:41 -0700
Subject: [PATCH] Export agave binaries during docker build (BP #627) (#628)

---
 dev/Dockerfile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dev/Dockerfile b/dev/Dockerfile
index bab9a1c02f..012ca45974 100644
--- a/dev/Dockerfile
+++ b/dev/Dockerfile
@@ -42,7 +42,7 @@ RUN --mount=type=cache,mode=0777,target=/solana/target \
     --mount=type=cache,mode=0777,target=/usr/local/cargo/registry \
     --mount=type=cache,mode=0777,target=/usr/local/cargo/git \
     if [ "$debug" = "false" ] ; then \
-      ./cargo stable build --release && cp target/release/solana* ./docker-output; \
+      ./cargo stable build --release && cp target/release/solana* ./docker-output && cp target/release/agave* ./docker-output; \
     else \
-      RUSTFLAGS='-g -C force-frame-pointers=yes' ./cargo stable build --release && cp target/release/solana* ./docker-output; \
+      RUSTFLAGS='-g -C force-frame-pointers=yes' ./cargo stable build --release && cp target/release/solana* ./docker-output && cp target/release/agave* ./docker-output; \
     fi
